python/cryptography/security/insecure-cipher-mode-ecb.yaml

```diff
@@ -30,3 +29,5 @@ rules:
   severity: WARNING
   languages:
   - python
+  pattern: cryptography.hazmat.primitives.ciphers.modes.ECB($IV)
+  fix: cryptography.hazmat.primitives.ciphers.modes.CBC($IV)
```

Comment:
We shouldn't recommend CBC since it has its own weaknesses as well (cf. all the padding oracles attacks). GCM is the recommended way nowadays. But then, you should move to using AHEAD.